sealtk/noaa/gui/Window.cpp

```diff
@@ -602,17 +577,35 @@ void WindowPrivate::createWindow(
   data->player->setDefaultColor(qRgb(240, 176, 48));
   data->createDetectionTool = new sg::CreateDetectionPlayerTool{data->player};
 
-  QObject::connect(q, &Window::zoomChanged,
-                   data->player, &sg::Player::setZoom);
-  QObject::connect(data->player, &sg::Player::zoomChanged,
-                   q, &Window::setZoom);
-  data->player->setZoom(q->zoom());
+  QObject::connect(
+    data->player, &sg::Player::zoomChanged, q,
+    [this, player = data->player](float zoom){
+      if (this->ui.actionViewLink->isChecked() && player->hasTransform())
```

Comment:
If this does what I think it does and is synchronizing the zoom across multiple viewpoints, perhaps `actionLinkViews` would be *slightly* clearer.

Reply:
Huh... I was going to mutter something about hierarchical naming, but nothing else is using that convention. Changed; thanks!